packages/amplify-java-runtime-provider/src/utils/package.ts

```diff
@@ -0,0 +1,31 @@
+import childProcess from 'child_process';
+import path from 'path';
+import { PackageRequest, PackageResult } from 'amplify-function-plugin-interface/src';
+
+export async function packageResource(request: PackageRequest, context: any): Promise<PackageResult> {
+  const resourceDir = path.join(request.srcRoot);
+  const meta = context.amplify.getProjectDetails().amplifyMeta;
```

Comment:
If you need access to the function name, we should update the PackageRequest. However, it doesn't seem like building / packaging should depend on the function name?